src/views/List.jsx

```diff
@@ -37,9 +40,18 @@ export function List({ data }) {
 				Hello from the <code>/list</code> page!
 			</p>
 			<ul>
-				{filteredItems.map((item) => (
-					<ListItem key={item.id} name={item.name} />
-				))}
+				{filteredItems.length > 0 ? (
```

Comment:
suggestion: pull out the check for an empty set of data and display the note + button conditionally - rather than nesting within a ternary. This will improve readability.

Reply:
We went ahead and removed the ternary and displayed the note and button with a short circuit operator.